src/packages/@ncigdc/modern_components/GenesBarChart/GenesBarChart.js

```diff
@@ -182,171 +184,51 @@ const Component = compose(
-        gain2:200,
-        gain1:1990,
-        loss1:220,
-        loss2:170,
-      },{  
-        symbol: "USH2A",
-        gene_id: "ENSG00000042781",
-        gain2:200,
-        gain1:150,
-        loss1:2000,
-        loss2:170,
-      },{  
-        symbol: "PCLO",
-        gene_id: "ENSG00000186472",
-        gain2:200,
-        gain1:150,
-        loss1:220,
-        loss2:1700,
-      },{  
-        symbol: "OBSCN",
-        gene_id: "ENSG00000154358",
-        gain2:200,
-        gain1:1500,
-        loss1:220,
-        loss2:170,
-      },{  
-        symbol: "MUC4",
-        gene_id: "ENSG00000145113",
-        gain2:2000,
-        gain1:1500,
-        loss1:220,
-        loss2:170,
-      },{  
-        symbol: "ZFHX4",
-        gene_id: "ENSG00000091656",
-        gain2:2000,
-        gain1:150,
-        loss1:2200,
-        loss2:170,
-      },{  
-        symbol: "DNAH5",
-        gene_id: "ENSG00000039139",
-        gain2:200,
-        gain1:150,
-        loss1:220,
-        loss2:170,
-      },{  
-        symbol: "CSMD1",
-        gene_id: "ENSG00000183117",
-        gain2:200,
-        gain1:150,
-        loss1:220,
-        loss2:170,
-      },{  
-        symbol: "XIRP2",
-        gene_id: "ENSG00000163092",
-        gain2:200,
-        gain1:150,
-        loss1:220,
-        loss2:170,
-      },{  
-        symbol: "DST",
-        gene_id: "ENSG00000151914",
-        gain2:200,
-        gain1:150,
-        loss1:220,
-        loss2:170,
-      },{  
-        symbol: "FAT3",
-        gene_id: "ENSG00000165323",
-        gain2:200,
-        gain1:150,
-        loss1:220,
-        loss2:170,
-      },{  
-        symbol: "FAT4",
-        gene_id: "ENSG00000196159",
-        gain2:200,
-        gain1:150,
-        loss1:220,
-        loss2:170,
-      }];
-    const cnaGenesChartData = cnaNodes
-      .sort((a, b) => checkers.reduce((acc, c) => b[c.key] - a[c.key] + acc, 0))
+    const cnvNodes = genes.hits.edges.map(x => ({
+      symbol: x.node.symbol,
+      gene_id: x.node.gene_id,
+      amplification: Math.round(x.node.score / 5),
+      gain: Math.floor(x.node.score / 5),
+      shallow_loss: Math.floor(x.node.score / 7),
+      deep_loss: Math.round(x.node.score / 7),
+    }));
+    const cnvGenesChartData = cnvNodes
+      .sort((a, b) =>
+        cnvColors.reduce((acc, c) => b[c.key] - a[c.key] + acc, 0),
+      )
       .map(g => {
-        const score = checkers.reduce((acc, c) => g[c.key] + acc, 0);
         return {
-          label: g.symbol,
-          value:
+          symbol: g.symbol,
+          deep_loss:
             context === 'project' && projectId
-              ? score / numCasesAggByProject[projectId] * 100
-              : score / filteredCases.hits.total * 100,
-          tooltip: tooltipContext(context, { symbol: g.symbol, score: score }),
-          onClick: () => handleClickGene(g, cnaGenesChartData),
+              ? g.deep_loss / numCasesAggByProject[projectId] * 100
```

Comment:
Can we pull these calculations into a helper function? Like:
 contextPercentageByCases = (change, aggs) => change/aggs
something like that?